sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -129,12 +130,42 @@ protected ServiceBusSender()
             CancellationToken cancellationToken = default)
         {
             Argument.AssertNotNull(message, nameof(message));
+            await SendAsync(
+                new ServiceBusMessage[] { message },
+                cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        ///   Sends a set of messages to the associated Service Bus entity using a batched approach.
+        ///   If the size of the messages exceed the maximum size of a single batch,
+        ///   an exception will be triggered and the send will fail. In order to ensure that the messages
+        ///   being sent will fit in a batch, use <see cref="SendBatchAsync"/> instead.
+        /// </summary>
+        ///
+        /// <param name="messages">The set of messages to send.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
+        ///
+        /// <returns>A task to be resolved on when the operation has completed.</returns>
+        ///
+        public virtual async Task SendAsync(
+            IEnumerable<ServiceBusMessage> messages,
+            CancellationToken cancellationToken = default)
```

Comment:
Here we are sending messages in a batch. Shouldn't this be called `SendBatchAsync`?

Reply:
Yes, the messages still get sent in a batch, but this is different from the SendBatchAsync method where the user creates a batch themselves. Technically, even the single send is sent in a batch.

Reply:
>Technically, even the single send is sent in a batch.

I think this is internal detail. 

Just like other methods when we deal with more than one messages we say ReceiveBatchAsync, PeekBatchAsync, ReceiveDeferredMessageBatchAsync... etc and when we deal with single message we say ReceiveAsync, PeekAsync, ReceiveDeferredMessageAsync.. etc. I think we should be consistent with other methods.

Reply:
I'm not sure about this. I like using SendAsync more for this as it should be a common use case. For the receive examples, the difference there is that the return type differs from the single Receive methods. Here the return is the same between send IEnumerable and send single message. Also the term batch is overloaded on the send side, because we actually have a way to send a ServiceBusMessageBatch. I think we can probably discuss this more before Preview 2 release.

Reply:
Yeah, lets discuss about this with other languages before release.

Reply:
> Just like other methods when we deal with more than one messages we say ReceiveBatchAsync, PeekBatchAsync, ReceiveDeferredMessageBatchAsync... etc and when we deal with single message we say ReceiveAsync, PeekAsync, ReceiveDeferredMessageAsync.. etc. I think we should be consistent with other methods.

Theoretically, you're correct it's a batch. But do you really want to confuse users with calling it a batch when there's an explicit batch API which is called `SendBatchAsync()`?

This is where I agree with @JoshLove-msft and don't think the method name should be mixed with something it is not. It's a collection of messages that are sent together, not a (safe) batch.